plugins/parallel.md

```diff
@@ -18,6 +19,16 @@ you can drastically reduce the time it takes to run large test suites.
 
 **Source code**: [github.com/pestphp/pest-plugin-parallel](https://github.com/pestphp/pest-plugin-parallel)
 
+<a name="requirements"></a>
+## Requirements
```

Comment:
I don't get this section. IMO this requirements are ensured by composer itself, via the conflict or requirements section.

Regarding Laravel, maybe throw an exception on the plugin itself if the version is not ^v8.55.0?

Reply:
Removed 👍